lib/cluster/membership.c

```diff
@@ -84,15 +84,27 @@ static gboolean crm_autoreap  = TRUE;
 static void update_peer_uname(crm_node_t *node, const char *uname);
 static crm_node_t *find_known_node(const char *id, const char *uname);
 
-int
-crm_remote_peer_cache_size(void)
+/*!
+ * \internal
+ * \brief Get the number of Pacemaker Remote nodes that have been seen
+ *
+ * \return Number of cached Pacemaker Remote nodes
+ */
+guint
```

Comment:
let's use unsigned int to minimize glib exposure. guint is guaranteed to be the same, so it won't hurt